### ISSUE TITLE
Scene Classification EuroSAT RGB or Multispectral?

### ISSUE THREAD

**Issue by ayush-AkulaTech** [ViTAE-Transformer/MTP#30]
Hi,
Just wondering if the fine-tuning on the EuroSAT dataset is using the RGB variant or the "all" variant of EuroSAT which contained the 13 spectral channels.

If RGB, is it easily compatible to fine-tune further/inference on multispectral data and see how it performs? Thanks!

**Comment by DotWang** [comment]
@ayush-AkulaTech we use RGB,

It can be finetuned on multispectral, after the dataloader and number of input channels changed

**Comment by ayush-AkulaTech** [comment]
@DotWang Thank you for your reply, I have another issue for the scene classification models:

It looks like the output predictions saved in the --show-dir directory are in BGR format. I’m puzzled by the to_rgb=True option in the data preprocessing section of the config. Changing it to False or True doesn’t seem to affect the final output format, but I noticed that setting it to False significantly drops the model's accuracy.

Could you clarify what format is actually used for saving the predictions? Does it use OpenCV or PIL? Additionally, since the dataset is stored in RGB format, why is there a to_rgb flag in the config? If LoadImageFromDisk uses OpenCV, wouldn’t it interpret the images as BGR without flipping the channels? I’d appreciate any clarification on these points!

**Comment by DotWang** [comment]
@DotWang the process you mentioned is control by the mmcv, but I'm curious about why the scene classification can produce a result map?

**Comment by ayush-AkulaTech** [comment]
@DotWang it results in an output which is the image (in BGR) with the prediction and ground truth label, not necessarily a "result map" if that's what you mean. Also as a side note, just wondering if you've tried exporting the model in ONNX format? Thanks!

**Comment by DotWang** [comment]
@ayush-AkulaTech So you are doing semantic segmentation instead of scene classification, right? We have not tried exporting the model in ONNX format. In my idea, the visualization of segmentation is controled by the `SegLocalVisualizer` function in mmsegmentation.
